.github/workflows/safety.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v3
       - uses: actions/setup-python@v5
+        with:
+          cache: 'pip'
```

Comment:
`uv` cache here too (see previous comment).

Reply:
Fixed - Thank you!!